### ISSUE TITLE
Learning rate scheduler doesn't match as stated in the paper.

### ISSUE THREAD

**Issue by v-trungdt21** [deepcam-cn/FaceQuality#10]
In the paper, you said that it would be decayed by 10 after 30, 60, 90 epoch for total of 100 epochs. But in the code, I saw that you were using CosineAnnealingRate, which doesn't have the effect as the above.
And also, I saw that u pass T_max hard-code 10 epochs (10 * len(train_loader)) -> is this intentional? Cause this would make the LR varies in a cyclical way.
Thank you for reading and answering.
